src/engine.rs

```diff
@@ -768,6 +768,8 @@ impl Reedline {
 
         let mut terminal_size = terminal::size()?;
 
+        self.stdout.queue(Print("\r\n"))?.flush()?;
```

Comment:
Do we always want to send a crlf? I was wondering if we may just want to see that we're at the bottom of the screen and scroll the screen up a line if so.